### PR TITLE
Update Readme with slack, twitter and logo

### PR DESCRIPTION
**Pull request by Better-Boy**
New Readme Look:

Screenshot 2020-10-10 at 12 06 04 PM

Added Get Help with slack and twitter links
Added link in readme to contributing.md file
Fixed the link in contributing.md file to find all chec projects

### CODE REVIEW

**Comment by Better-Boy** [comment]
@robbieaverill Can you please review this PR?

File: README.md [hunk 1]
```diff
@@ -48,6 +65,11 @@ The `cart.add()` method now separates its arguments:
 ```
 
 ## Contributing
+Check out the [contributing guide](CONTRIBUTING.md)
+
+### Get Help
+
+Commerce.js is a project by [many contributors](https://github.com/chec/commerce.js/graphs/contributors). Reach us on [slack](http://slack.commercejs.com) and [twitter](https://twitter.com/commercejs)
```
Comment (robbieaverill) [inline README.md:72]:
```suggestion
Commerce.js is a project by [many contributors](https://github.com/chec/commerce.js/graphs/contributors). Reach us on [Slack](http://slack.commercejs.com) and [Twitter](https://twitter.com/commercejs).
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,13 +9,20 @@ The SDK documentation module source code resides in `commerce.js/docs`. The docs
...
   - [Edit online](#edit-online)
   - [Commit messages and branch name](#commit-messages-and-branch-name)
   - [Edit locally](#edit-locally)
   - [Contributing to Chec open source](#contributing-to-other-chec-open-source-projects)
 
+## Get Help
+Do reach out on Slack or Twitter and we are happy to help.
+
+1. Drop by the [slack channel](http://slack.commercejs.com).
+2. Say hi on [twitter](https://twitter.com/commercejs).
```
Comment (robbieaverill) [inline CONTRIBUTING.md:24]:
```suggestion
1. Drop by the [Slack channel](http://slack.commercejs.com).
2. Say hi on [Twitter](https://twitter.com/commercejs).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -9,13 +9,20 @@ The SDK documentation module source code resides in `commerce.js/docs`. The docs
 - Errors, typos, or any other improvements to current documentation
 - A full-page new documentation to highlight a feature or resource
 
+
```
Comment (robbieaverill) [inline CONTRIBUTING.md:12]:
```suggestion
```

**Review by robbieaverill — COMMENTED** [review]
Thank you, I've made some minor suggested edits

**Review by robbieaverill — APPROVED** [review]

**Pull request merged**